### DISCUSSION TITLE
Subir los archivos directo en el Repositorio

### DISCUSSION THREAD

**Discussion by Gracesl** [LaunchX-InnovaccionVirtual/CursoIntroPython#322] [Q&A]
me sale este error del remote origin
![errorsubirkatasgitbash](https://user-images.githubusercontent.com/82528455/154854257-3280b7d0-0617-43ad-afca-f2d6757807f2.png)
Ya lo logré finalmente arrastre directamente los archivos

**Comment by JuanSalvador73** [comment]
Lo que yo hice fué arrastrar los archivos directamente desde ventanas en windows, como la opción que te da la plataforma. Más rápido y sencillo.

**Comment by doguedogue** [comment]
siempre puedes ver lo que configuraste como remoto con 
`git remote -v`
también puedes cambiarlo/renombrarlo, ese fue tu error al añadirlo nuevamente
lo que seguía era hacer el PUSH
normalmente estos son los pasos:
```
git remote add origin XXXXXXX.git
git branch -M main
git push -u origin main
```
